package.json

```diff
@@ -17,7 +17,7 @@
         "start:dev": "cross-env USE_DEV_DB=true webpack serve --config ./webpack/webpack.config.js  --env env=development",
         "lint": "eslint src/**/*.{ts,tsx}",
         "typeCheck": "tsc -p tsconfig.json --noEmit",
-        "test": "cross-env BABEL_ENV=commonjs TS_NODE_FILES=true TS_NODE_PROJECT=tsconfig.commonjs.json NODE_ENV=production mocha src/**/*.test.ts",
+        "test": "cross-env vitest",
```

Comment:
Should be no need for cross-env here, there's no setting of environment variables